identity/did/service.go

```diff
@@ -14,17 +14,17 @@ var log = logging.Logger("identity")
 
 // Service is the interface for identity related interactions
 type Service interface {
-	CreateIdentity(ctx context.Context) (id *DID, confirmations chan *identity.WatchIdentity, err error)
+	CreateIdentity(ctx context.Context) (id *DID, confirmations chan *id.WatchIdentity, err error)
```

Comment:
Shouldn't we implement as well `FindIdentity` function? Or that would be done after? I think it makes sense to have added it when creating the identity was done, or now.

Reply:
I implemented this method:

```golang
func (i identity) bindContract(did DID) (contract, error) {
	contract, err := NewIdentityContract(did.toAddress(), i.client.GetEthClient())
	if err != nil {
		return nil, errors.New("Could not bind identity contract: %v", err)
	}

 	return contract, nil

 }
```

The user needs to provide the DID for every identity interaction. The first thing every method will do is to bind the provided DID with the contract. I could add an additional check by getting the byteCode of the contract to check if it is an identity contract.

Reply:
That is fine for now, Can we then create a public method called FindIdentity that invokes `bindContract` internally and returns the identity object?